apis/v1alpha1/generator.yaml

```diff
@@ -0,0 +1,8 @@
+ignore:
```

Comment:
A controller without any resources breaks some of our CI/CD, but I can't remember exactly where. I'll have to keep an eye on this as it goes through

Reply:
This was per @A-Hilaly request to exclude all resources for the scaffolding PR and add each resource afterwards. I have a fully working `generator.yaml` in [my branch](https://github.com/embano1/eventbridge-controller/blob/aminemicha/generator.yaml), not sure which fields you want me to add to this base?

Reply:
Sorry, my bad and misunderstanding. Updated the file